driver/src/main/java/oracle/nosql/driver/ops/QueryRequest.java

```diff
@@ -15,13 +15,16 @@
 
 import oracle.nosql.driver.Consistency;
 import oracle.nosql.driver.Durability;
+import oracle.nosql.driver.JsonParseException;
```

Comment:
is this being used? We don't have a handy warning target to check for unused imports. Does your IDE do it? This applies to other files as well